src/pipeline/blitter.rs

```diff
@@ -18,13 +18,13 @@ use crate::pixmap::SubPixmapMut;
 
 pub struct RasterPipelineBlitter<'a, 'b: 'a> {
     mask: Option<SubMaskRef<'a>>,
-    pixmap_src: PixmapRef<'a>,
     pixmap: &'a mut SubPixmapMut<'b>,
     memset2d_color: Option<PremultipliedColorU8>,
     blit_anti_h_rp: RasterPipeline,
     blit_rect_rp: RasterPipeline,
     blit_mask_rp: RasterPipeline,
     is_mask: bool,
+    mipmaps: Mipmaps<'a>,
```

Comment:
I think it is better to keep the `mipmaps` member up above probably?

Reply:
I've moved it to where `pixmap_src` was, if that seems reasonable?